### ISSUE TITLE
[qa] about global functions

### ISSUE THREAD

**Issue by lichunqiang** [samdark/yii2-cookbook#89]
as you said:
```
function t($message, $params = [], $category = 'app', $language = null)
{
    return Yii::t($category, $message, $params, $language);
}
```
this is shorthand function for `Yii::t`. 

My question is how to make `yii/message` to meet this function.

I tried and found that, the `yii/message` will extract the message as `Yii::t` defined params order, so If i changed the order of params the `yii/message` can not work well.

Any suggestion?

**Comment by samdark** [comment]
Yes, I think it's something it could not handle now. Needs to be improved...

**Comment by lichunqiang** [comment]
Yii framework can handle this scenario will be nice.

**Comment by samdark** [comment]
Absolutely.
